### DISCUSSION TITLE
Feature Request: Windows 10 Task View (Timeline)

### DISCUSSION THREAD

**Discussion by XuanChen1234** [valinet/ExplorerPatcher#1664] [Ideas]
We are in the task view, they are Windows 11 by default. If it can be like task view in windows 10, so much the better.

Current Windows 11 Task View:
![屏幕截图 2023-07-12 143511](https://github.com/valinet/ExplorerPatcher/assets/140480110/4c0380b8-d836-4260-af6a-6b46ec59b695)

Windows 10 Task View, Show Timeline:
![image](https://github.com/valinet/ExplorerPatcher/assets/140480110/451a997b-6659-45e4-8248-4f30ea492b89)

Is there anyway to add W10 task view, or please rollout an updates for this, Thanks @valinet !

**Comment by XuanChen1234** [comment]
@valinet

**Comment by XuanChen1234** [comment]
If it is not planned yet, I hope this will be a thing in the next update.

**Comment by pyrates999** [comment]
I would say this is a no.

**Reply by XuanChen1234** [reply]
Ask for the option to add Windows 10 Task View!
I'm on version 22631.3007!
![image](https://github.com/valinet/ExplorerPatcher/assets/140480110/6acd6abb-81aa-4b6c-b14b-d20986d42fbd)

**Reply by XuanChen1234** [reply]
@valinet

**Comment by XuanChen1234** [comment]
@LforGithub Any plans for the Windows 10 Task View?

**Reply by milestprower92** [reply]
Not possible

**Comment by XuanChen1234** [comment]
@Vipin2003 Any plans for the Windows 10 Task View options?

**Reply by XuanChen1234** [reply]
@lingbing76

**Comment by XuanChen1234** [comment]
@Vipin2003 Ask for the option to add Windows 10 Task View!!!

**Reply by milestprower92** [reply]
How many time we have to say it

IT'S NOT POSSIBLE!!!!!

**Reply by XuanChen1234** [reply]
Ask for the option to add Windows 10 Task View!!! (Windows 10 작업 보기를 추가하는 옵션을 요청하세요!!)

**Reply by XuanChen1234** [reply]
Windows 10 작업 보기를 추가하는 옵션을 요청하세요!

**Comment by XuanChen1234** [comment]
@valinet @Vipin2003 Ask for the option to add Windows 10's task view!

**Reply by vippium** [reply]
It is not...bcoz in first release of w11, it is not present. MS changed it completely. I think that porting it not possible bcoz of messing code. First priority is to make EP deal with upcoming 24h2 with so many changes

**Reply by vippium** [reply]
You can ask from @Amrsatrio

**Reply by Amrsatrio** [reply]
No.

**Reply by pyrates999** [reply]
This should be closed since the feature request has been declined.

**Comment by XuanChen1234** [comment]
@valinet @Vipin2003 Ask for the option to add Windows 10 Task View!!!

**Comment by XuanChen1234** [comment]
@coraphise Ask for the option to add Windows 10's task view!

**Comment by XuanChen1234** [comment]
@valinet Ask for the option to add Windows 10 Task View!
